bluebanquise-bootset/bootset

```diff
@@ -39,11 +39,20 @@ class bcolors:
 
 def load_file(filename):
     logging.info(bcolors.OKBLUE+'Loading '+filename+bcolors.ENDC)
+
+    # Select YAML loader (needs PyYAML 5.1+ to be safe)
+    yaml_version_major=int(yaml.__version__.split('.')[0])
+    yaml_version_minor=int(yaml.__version__.split('.')[1])
+    if yaml_version_major > 5 or (yaml_version_major == 5 and yaml_version_minor >= 1):
+        yaml_loader='Safe'
+    else:
+        yaml_loader='Unsafe'
```

Comment:
Could use a Boolean instead to avoid hardcode strings:

```
yaml_loader_safe = yaml_version_major > 5 or (yaml_version_major == 5 and yaml_version_minor >= 1)
```

Reply:
I didn't know you could do that ! Ok, I will update the PR.

Reply:
Yes, you can assign the output of a logical operation to a variable :-)